api/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM ubuntu:latest
+ENV SERVER_HOST=0.0.0.0
+RUN apt-get update -y
```

Comment:
Combine this line with the next see the [best practices docs](https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#/run) for more info.